client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/SubmissionEditStepForm.jsx

```diff
@@ -422,13 +422,25 @@ class SubmissionEditStepForm extends Component {
         style={{ justifyContent: 'center', flexWrap: 'wrap' }}
       >
         {questionIds.map((questionId, index) => {
+          let stepButtonColor = '';
+          if (index === stepIndex) {
```

Comment:
I think I'll personally simplify this to:

```js
const isCurrentQuestion = index === stepIndex;
if (explanations[questionId] && explanations[questionId].correct) {
  stepButtonColor = isCurrentQuestion ? green900 : green500;
} else {
  stepButtonColor = isCurrentQuestion ? blue800 : lightBlue400;
}
```

Reply:
Noted and thanks!!